CHANGELOG.rst

```diff
@@ -5,6 +5,8 @@
 New Features
 ------------
 
+* junitxml: Add `--junit-suite-name` option to specify root `<testsuite>` name for JUnit XML reports
```

Comment:
This is a RST file, so you should use double back-ticks to get a monospaced font:

```rst
* junitxml: Add ``--junit-suite-name`` option to specify root ``<testsuite>`` name for JUnit XML reports.
```